src/chocolatey/infrastructure.app/configuration/ConfigurationOptions.cs

```diff
@@ -75,8 +75,11 @@ public static OptionSet OptionSet
             {
                 _optionSet
                     .Add("?|help|h",
-                         "Prints out the help menu.",
-                         option => configuration.HelpRequested = option != null);
+                        "Prints out the help menu.",
+                        option => configuration.HelpRequested = option != null)
+                    .Add("online",
+                        "Online - Open help for specified command in default browser application.",

```

Comment:
Should there be a mention about `--online` can only be used when `--help` or `-h` is used?

Reply:
That is a good suggestion, I will get that added.

Reply:
Would it make more sense to have `--online` imply `--help` and just make it clear in the output that it will open the help for the command online?

Reply:
No, I don't think so.

This addition, is the first of a few things that I would like to do in this area.  For example, I would like to be able to do:

```
choco install -h --examples
```

and

```
choco install -h --detailed
```

etc.  So I don't think having it imply the help option is a good idea.